13_the_final_evening.po

```diff
@@ -380,7 +380,7 @@ msgstr ""
 "Una pequeña sombra de extremidades múltiples salta hacia ti cuando te acercas a Judith. Antes de que puedas reaccionar, se escucha un disparo y ves lo que parece un puercoespín cruzado con un carcayú desplomarse en un charco de sangre. Judith avanza y mira el cadáver con el fusil humeante en la mano.\n"
 "–El viejo Jomo ha vuelto a salvarle el pellejo –dice mientras se echa el fusil al hombro–. Mi contrato se acaba después de hoy. En cuanto finalice, me largo de este sitio –Te mira de arriba a abajo y añade–. Espero que ustedes también."
 
-msgid "Increase Judith Park’s Relationship Level by 2. Each investigator earns 2 bonus experience."
+msgid "Increase Judith Park’s Relationship Level. Each investigator earns 2 bonus experience."
```

Comment:
changed all keys by hand tbh to not break existing po files. afaik the translators took the correct value already (as far as I checked with german, chinese, korean, french, ...)